src/logger/loggerApi.c

```diff
@@ -1837,3 +1838,35 @@ int api_set_telemetry_stop(struct Serial *serial, const jsmntok_t *json)
                 serial_ioctl(serial, SERIAL_IOCTL_TELEMETRY_DISABLE, NULL);
         return get_telemetry_api_return_code(cmd_result);
 }
+
+int api_set_active_track(struct Serial *serial, const jsmntok_t *json)
+{
+        Track track;
+        float tmp;
+        float radius_m;
+
+        const jsmntok_t *json_track = jsmn_find_node(json, "track");
+        /* If no track is given, then fail */
+        if (json_track == NULL)
+                return API_ERROR_PARAMETER;
+
+        memset(&track, 0, sizeof(track));
+        setTrack(json_track + 1, &track);
+
+        const bool rad_set = setFloatValueIfExists(json, "rad", &tmp);
+        const bool radius_set = setFloatValueIfExists(json, "radius", &radius_m);
```

Comment:
Yes, but with full backwards compatibility.  In traditional track setting method the "rad" value holds the value of radius in degrees of latitude.  This is still true here too.  The "radius" variable is new and takes the radius value in meters.  Its value trumps that of "rad" if set.

Internally we have been converting that value to meters since 2.8.0 days (whenever I did my first major refactor on lap stats).  The only reason it has stayed around is because the API supports it that way (but I imagine you never set/use it in the App).